src/delaunay.js

```diff
@@ -50,20 +50,20 @@ export default class Delaunay {
       }
       d = new Delaunator(points);
     }
-    this.delaunator = d;
+    this._delaunator = d;
+    this._init();
+  }
+  update() {
+    this._delaunator.update();
     this._init();
-    this.update = () => {
-      this.delaunator.update();
-      this._init();
-    }
   }
   _init() {
-    const points = this.points = this.delaunator.coords;
-    const halfedges = this.halfedges = this.delaunator.halfedges;
-    const hull = this.hull = this.delaunator.hull;
-    const triangles = this.triangles = this.delaunator.triangles;
+    const points = this.points = this._delaunator.coords;
+    const halfedges = this.halfedges = this._delaunator.halfedges;
+    const hull = this.hull = this._delaunator.hull;
+    const triangles = this.triangles = this._delaunator.triangles;
     const inedges = this.inedges = new Int32Array(points.length / 2).fill(-1);
-    this._hullIndex = new Int32Array(points.length / 2).fill(-1);
+    const hullIndex = this._hullIndex = new Int32Array(points.length / 2).fill(-1);
```

Comment:
One more: `inedges` and `hullIndex` don't have to be allocated anew on `update` since their length doesn't change. It's enough to do `fill(-1)` on existing arrays.